CMakeLists.txt

```diff
@@ -242,9 +242,12 @@ endif()
 if (MFEM_USE_SUNDIALS)
   if (NOT MFEM_USE_MPI)
     find_package(SUNDIALS REQUIRED NVector_Serial CVODES ARKODE KINSOL)
+  elseif(NOT MFEM_USE_CUDA)
+    find_package(SUNDIALS REQUIRED
+      NVector_Serial NVector_Parallel NVector_MPIPlusX CVODES ARKODE KINSOL)
   else()
     find_package(SUNDIALS REQUIRED
-      NVector_Serial NVector_Parallel NVector_ParHyp CVODES ARKODE KINSOL)
+      NVector_Serial NVector_Cuda NVector_Parallel NVector_MPIPlusX CVODES ARKODE KINSOL)
   endif()
```

Comment:
The case of `MFEM_USE_CUDA=ON MFEM_USE_MPI=OFF` is not handled correctly. Maybe defining a variable for which `NVector_*` components to require can simplify the logic.

Reply:
Oops... fix incoming.